src/models/tiles/geometry/geometry-content.ts

```diff
@@ -192,6 +194,26 @@ export const GeometryContentModel = GeometryBaseContentModel
     }
   }))
   .views(self => ({
+    get annotatableObjects() {
+      const tileId = getTileIdFromContent(self) ?? "";
+      const polygons: IClueObject[] = [];
+      const segments: IClueObject[] = [];
+      const points: IClueObject[] = [];
```

Comment:
There are, of course, other objects that could be linked: vertex angles, comments, potentially background images, axes, etc., but those can wait until somebody requests them.